containers/concent-api/install-dependencies.sh

```diff
@@ -2,22 +2,35 @@
 
 # Update package lists for Debian and install dependencies
 dependencies=(
-    python3-pip
-    python3-virtualenv
-    virtualenv
-
     # Required by psycopg2 python package (PostgreSQL support)
     libpq-dev
 
     # Packages needed to build python extensions, dependencies of golem-messages in particular
     gcc
-    python3-dev
-    libssl-dev
+    libssl1.0.2
+    libssl1.0-dev
     git
+    pkg-config
+    libsecp256k1-dev
+    libffi-dev
+    software-properties-common
+)
+
+# We need to use python3.6. There is no official python3.6 package on the debian stretch,
+# we must include test packages.
+python_dependencies=(
+    python3.6
+    python3.6-dev
+    python3.6-venv
 )
+
 apt-get --assume-yes update
 apt-get --assume-yes install --no-install-recommends ${dependencies[*]}
+add-apt-repository "deb http://ftp.de.debian.org/debian testing main"
+apt-get --assume-yes update
+apt-get --target-release testing --assume-yes install --no-install-recommends ${python_dependencies[*]}
```

Comment:
Move the `add-apt-repository` line above the first `apt-get update`. This way yuo won't have to update twice.

Reply:
The `software-properties-common` package allows to use `add-apt-repository`, which is in dependencies, so before we can use `add-apt-repository`, we must install depedencies.